ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -55,10 +63,13 @@ type RunParams struct {
 	NetworkConfiguration NetworkConfiguration `yaml:"network_configuration"`
 }
 
+// NetworkConfiguration specifies the nertwork config for the task definition
```

Comment:
`nertwork`!
Maybe we can add the supported values? `awsvpc`, `host`, `bridge` or `none` (default bridge if left blank IIRC). Also we could add that it's required for Fargate?